setup.py

```diff
@@ -22,7 +22,7 @@
 name = "google-cloud-dlp"
 description = "Cloud Data Loss Prevention (DLP) API API client library"
 version = "0.13.0"
```

Comment:
Needs a bump to 1.0.0?

Reply:
Release-please will bump the release version string.

There's a new feature in the v1 though. See pending release in #14 and https://github.com/googleapis/python-dlp/commit/ffad36ec37e62648f81830ecabbccb1d57e49036. Does that reset the bake time?

Reply:
We don't typically annotate methods/rpcs as beta within a GA client do we? I guess this should reset the clock.